### ISSUE TITLE
Vacation Policy

### ISSUE THREAD

**Issue by chawang90** [josephine/handbook#12]
B told me that the hardest thing for them to agree on when writing the Clef Handbook was [vacation policy](https://github.com/josephine/handbook/blob/master/Benefits%20and%20Perks/Vacation%20and%20Sick%20Leave.md). In the end they decided to set more structure in the handbook and then deal with exceptions to the rule as they came up. 

I'm torn between this and something a little more idealistic. Ie. unlimited vacation with a 15 day minimum (really like the idea of minimum PTO since a lot of startups give unlimited vacation w/ no minimum as a way of pressuring people to work more and not having accrual of days off). 

However, as soon as we set unlimited vacation then that makes it really hard for us to distinguish between sick days, PTO, sabbatical, remote days, holidays, and other protected absences. 

Would love feedback on this, part of me is thinking maybe we articulate our ideals and hopes (ie. everyone should be here for the right reasons and we trust our culture to create responsible decision making) but also caveat that for pragmatic reasons we've decided to set a structure (maybe a minimum and a maximum for PTO? ie. 15-20 days?)

As much as I'd love to be able to say we have an unlimited vacation policy, I feel that it might be too complicated to start with, given that we'd have to come up with nuanced definitions to ensure that our other types of  protected absences are enforceable.

**Comment by joyjding** [comment]
I think that 15-20 days makes sense (and is generous) as long as there
isn't any shame attached to taking more/less in that range.

**Comment by zeke** [comment]
Heroku has unlimited PTO. It was great, and I never knew anyone to abuse it. I hear anecdotally that people get socially pressured to not take time off, but that hasn't been my experience. I think PTO works, and everyone's tendency is to be considerate of the impact their absence will have on the team.

**Comment by chawang90** [comment]
@zeke were there other types of paid absence that were counted? Ie. Sick days, remote work days, new parent leave, etc?

**Comment by joyjding** [comment]
After reading Zeke's answer, and spending more time at Josephine, I think that keeping unlimited vacation makes sense. I trust all of us to be responsible and considerate.

**Comment by sike13** [comment]
Me and @talsafran talked about this briefly right when I got back from Ghana. I think if we don't do unlimited vacation, counting absences gets tricky/cumbersome. How would you count a day that someone worked all day but remotely (e.g. when I was in NY and wiki-ing)? Would that be vacation time or does that count as a regular work day? 

The main reason I say this is that any vacation I take to see my family takes a long time because of the distance. I'm okay with knowing that I can take fewer/longer vacations but it wouldn't be ideal to not take time off if I needed it because I was worried about having enough time to go see my family at a later date.

TL;DR — if we don't have an unlimited vacation policy, we need to make sure that how we distinguish vacation from remote days etc. is really clear from the outset.

**Comment by egustafson5** [comment]
I think a more generous policy is fitting for us (give boldly, give first), and we can reserve the right to reassess on an individual level and on a company level if things aren't going well. I personally really enjoy the level of trust this team supports. It very much decreases work-related stress.

**Comment by terrybetts24** [comment]
As a team we have a track record of respect for time off so unlimited time off seems the right choice. I do agree with SIka that it would be good to differentiate WFH and PTO to add additional flexibility for taking care of yourself or your family but still being connected to the team.
